test/Drips.t.sol

```diff
@@ -263,35 +263,58 @@ contract DripsTest is Test, PseudoRandomUtils, Drips {
         }
     }
 
+    function drainBalance(uint256 userId, uint128 balanceFrom) internal {
+        setDrips(userId, balanceFrom, 0, loadCurrReceivers(userId), 0);
+    }
+
     function setDrips(
         uint256 userId,
         uint128 balanceFrom,
         uint128 balanceTo,
-        DripsReceiver[] memory newReceivers
+        DripsReceiver[] memory newReceivers,
+        uint256 expectedMaxEndFromNow
+    ) internal {
+        setDrips(userId, balanceFrom, balanceTo, newReceivers, 0, 0, expectedMaxEndFromNow);
```

Comment:
Would it not make sense if the `expectedMaxEndFromNow` is known to pass two tips?
`expectedMaxEndFromNow, expectedMaxEndFromNow+1`?
(Assuming here expectedMaxEndFromNow last timestamp with enough balance)

Reply:
I mean it is a test so doesn't really matter :) If we don't test the `tips` here in specific

Reply:
Yeah, the gas savings aren't important, it's much more important to keep the code simple. Without tips the tests are also more thorough, because the max end search must be done over the entire space, so the code coverage should be a bit better, and more edge cases may be caught.